app/controllers/locations_controller.rb

```diff
@@ -0,0 +1,5 @@
+class LocationsController < ApplicationController
+  def index
+    @locations = Locations.near_postcode(params['postcode'])
```

Comment:
I originally had that function return some static data, I have it returning an empty array now until we get some specs in place